<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>perspectiveTransformation</class>
 <widget class="QDialog" name="perspectiveTransformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="matDisplay" name="labelPersTrans">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>531</width>
     <height>351</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>lapangan.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_P1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P1 (not set)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_P2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P2 (not set)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_P3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P3 (not set)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_P4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P4 (not set)</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>matDisplay</class>
   <extends>QLabel</extends>
   <header location="global">matdisplay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>perspectiveTransformation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>perspectiveTransformation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
